src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import ColorPickerPalette from './ColorPickerPalette';
+import { AnnotationMode } from '../annotations/types';
+import './ColorPickerControl.scss';
+
+export type Props = {
+    annotationMode?: AnnotationMode;
+    onAnnotationColorClick: (color: string) => void;
+    isActive?: boolean;
+};
+
+export default function ColorPickerControl({
+    annotationMode,
+    isActive = false,
+    onAnnotationColorClick,
+    ...rest
+}: Props): JSX.Element | null {
+    const [isColorPickerToggled, setIsColorPickerToggled] = useState(false);
+
+    if (annotationMode !== AnnotationMode.DRAWING) {
```

Comment:
Does this picker need to know about annotations at all? Could the rendering of this `ColorPickerControl` be controlled by the consuming component?